README.md

```diff
@@ -42,26 +42,26 @@ Please see
 
```

Comment:
Otherwise this is a good introduction. If you have time, a diagram of how the server/client/backends sit within the different clusters would be helpful